source/voice_control/index.markdown

```diff
@@ -1,82 +1,53 @@
 ---
-title: Assist - Talking to Home Assistant
+title: Talking with Home Assistant - get your system up & running
 related:
   - docs: /voice_control/android
     title: Assist on Android
-  - docs: /voice_control/android/#assist-on-wear-os
-    title: Assist on Wear OS 
   - docs: /voice_control/apple
-    title: Siri and Assist shortcuts
-  - docs: /voice_control/start_assist_from_dashboard/
-    title: Assist dashboard button
+    title: Assist on Apple
   - docs: /voice_control/thirteen-usd-voice-remote/
     title: Build a 13$ voice remote using an ESPHome device
-  - docs: /voice_control/install_wake_word_add_on
-    title: Enable a wake word
-  - docs: /voice_control/create_wake_word/
-    title: Create your own wake words
-  - docs: /voice_control/builtin_sentences
-    title: Sentences starter kit
+  - docs: /voice_control/best_practices/
+    title: Best practices with Assist
   - url: https://www.nabucasa.com/config/
     title: Home Assistant Cloud
 ---
 
-<img src='/images/assist/assist-logo.png' class='no-shadow' alt='Assist logo' style='width: 150px; float: right'>
+This section will help you set up Assist, which is Home Assistant voice assistant. 
 
-Assist allows you to control Home Assistant using natural language. It is built on top of an open voice foundation and powered by knowledge provided by our community.
+Assist allows you to control Home Assistant using natural language. It is built on top of an open voice foundation and powered by the knowledge provided by our community.
 
-Assist is available to use on most platforms that can interface with Home Assistant. Look for the Assist icon <img src='/images/assist/assist-icon.svg' alt='Assist icon' style='height: 32px' class='no-shadow'>:
+As for the rest of Home Assistant core functionalities, Assist can be personalized and extended to fit your needs.
+- It can work locally or leverage the greatest LLMs of the moment.
+- It can work on your phone or tablet or other custom voice devices.
 
 <lite-youtube videoid="XF53wUbeLxA" videotitle="Voice at Home Assistant"></lite-youtube>
 
-_Want to use Home Assistant with Google Assistant or Amazon Alexa? Get started with [Home Assistant Cloud](https://www.nabucasa.com/config/)._
+Although adding voice to your smart home configuration is exciting, it will require you check your existing setup of Home Assistant, specially if you made a lot of customization. But we have prepared a guide of steps and best practices to help you out and our [Troubleshooting](/voice_source/troubleshooting/) guides.
```

Comment:
```suggestion
Although adding voice to your smart home configuration is exciting, it will require you to check your existing setup of Home Assistant, especially if you made a lot of customization. But we have prepared a guide of steps and best practices to help you out, as well as our [Troubleshooting](/voice_source/troubleshooting/) guides.
```